### ISSUE TITLE
VB :: tinyMCE error appears in some cases

### ISSUE THREAD

**Issue by SlavaET** [elegantthemes/Divi-Beta#867]
### Problem:
It's quite inconsistent, but I can catch it:
![tinymce-error](https://cloud.githubusercontent.com/assets/3446682/21514880/789e1188-ccf4-11e6-9b89-729298798dec.gif)

### Steps To Reproduce:
1. Toggle accordion items and switch between inline editors on page. Error will appear at some point.

### Attached PR
https://github.com/elegantthemes/submodule-builder/issues/1549

**Comment by SlavaET** [comment]
I found a way to reproduce this error quite consistently:
1) Toggle accordion item;
2) Quicly click in the opening toggle item content to activate the inline editor before the toggling animation is finished;
3) Click any button in opened inline editor popover or in some other module to activate the inline editor in different place.
![accordion-tinymce-error](https://cloud.githubusercontent.com/assets/3446682/21538012/86d4e898-cdc2-11e6-810e-4af47afee515.gif)

As far as I understand this error occurs because inline editor is activate too early and when the toggling animation is finished we re-render the accordion module. At that point inline editor loses the original module where it was opened and it leads to an error when you try using it or deactivating it.
I added mechanism which prevents inline-editor from activating if module is not ready yet and implemented it for accordion ( this error doesn't occur in any other module as far as I can tell, even in Toggle module it works fine ).
So basically we just wait until the animation is finished and then allow to activate the inline editor disabling the ability to activate it during the module animation.

**Comment by lots0logs** [comment]
@SlavaET The accordion's animation is not right on this branch:

#### This Branch:
![2016-12-29 15-32](https://cloud.githubusercontent.com/assets/4675662/21555284/427c6c06-cddc-11e6-984b-f71f1e37be8a.gif)

#### Master:
![2016-12-29 15-32-2](https://cloud.githubusercontent.com/assets/4675662/21555288/4c63f3e2-cddc-11e6-961c-5041ca3614dc.gif)

**Comment by SlavaET** [comment]
@lots0logs fixed. Actually I fixed the original issue in a different way and it's possible to activate tinyMCE while toggling accordion now.
